examples/typescript/platform-ref-aws/crossplane.ts

```diff
@@ -0,0 +1,125 @@
+import { App, Chart } from 'cdk8s';
+
+import { Configuration, CompositeResourceDefinition, Prop, InputType } from '../../../src';
+
+import * as xrds from './crs/xrd.apiextensions.crossplane.io';
+import * as comp from './crs/composition.apiextensions.crossplane.io';
+
+export function configYaml(crossplanePackage: App): Chart {
+
+  const yaml = new Chart(crossplanePackage, 'crossplane');
```

Comment:
It's more idiomatic to subclass Chart instead of using functions. This way, you can offer a richer api for consumers. 

Something like;

```ts
class AwsPlatformConfiguration extends Chart {
  ....
}
```

Then, the usage is simply:

```ts
new AwsPlatformConfiguration(app, 'config');
```

Reply:
great idea, will look at including this in my next pass.